include/SugarDateTime.php

```diff
@@ -327,8 +335,8 @@ public function get_day_by_index_this_week($day_index)
 
     /**
      * Get the beginning of the last day of i's the month
-     * @deprecated
-     * FIXME: no idea why this function exists and what's the use of it
+     *
+     * @deprecated This function is unused and will be removed in a future release.
```

Comment:
I actually removed this in #8128, should maybe just remove this part from this PR to avoid conflicts?